pkg/plugins/policies/core/rules/testdata/resourcerules/mretry_with_mes.zone-uni.golden.yaml

```diff
@@ -0,0 +1,73 @@
+mesh:name/mesh-1:
```

Comment:
do all of these differ in any way other than names? (I guess from syncing right?)

Reply:
Yes, it's mostly to check if everything still works after syncing and adding the hash. I've added another use case to the existing test base.